client/src/link_success.ts

```diff
@@ -12,10 +12,6 @@ function displaySuccessMessage(app: LinkExchangeResponse) {
         </div>
```

Comment:
Should we update the line just above this as well?

> The access token and patient ID are now stored in the AppContext.

Github isn't letting me propose a change on that line sadly